### PR TITLE
rhoai-4386 model serving and ui updates for 2.8

### PR DESCRIPTION
**Pull request by MelissaFlinn**
updating the Fraud detection tutorial files to address model serving and UI updates in RHOAI 2.8
see https://issues.redhat.com/browse/RHOAIENG-4386 for details

### CODE REVIEW

File: workshop/docs/modules/ROOT/pages/storing-data-with-data-connections.adoc [hunk 1]
```diff
@@ -6,12 +6,121 @@ For this {deliverable}, you need two S3-compatible object storage buckets, such
...
+
+* Creates a Minio instance in your project.
+* Creates two storage buckets in that Minio instance.
+* Generates a random user id and password for your Minio instance.
+* Creates two Data Connections in your data science project, one for each bucket and both using the same credentials.
+* Installs required network policies for service mesh functionality.
+
+This script is based on a https://ai-on-openshift.io/tools-and-applications/minio/minio/[guide for deploying Minio].
+
+IMPORTANT: The storage buckets that this script creates are *not* meant for production usage.
```
Comment (erwangranger) [inline workshop/docs/modules/ROOT/pages/storing-data-with-data-connections.adoc:21]: I'd re-write as 
```
The Minio-based Object Storage that this script creates is *not* meant for production usage
```

**Comment by erwangranger** [comment]
@MelissaFlinn , it's a bit hard to read through the changes without a rendered version, but this is a great update. 

I do wonder however about the model serving. I'd expect that Single and Multi  would end up creating API endpoints that are a bit different in the way their URLs are crafted and accessed. I looked in Notebook 3, but there only seem to be one request here, for the Multi model serving. 
We may need to update this too, to reflect how to query a model deployed in the single-model version. 

This should not delay the current effort, though. It can be done later. 
Thanks for this.

**Comment by MelissaFlinn** [comment]
re-doing this PR

**Comment by MelissaFlinn** [comment]
@Erwan - should I remove the module for single-model serving for now. I believe that Chris Chase is addressing that in his test drive update.

**Comment by erwangranger** [comment]
I think having both options is great to have. People can choose which one they want. 
But the testing/validation of the endpoint will be slightly different for each, so we'll just need to add a matching notebook. 
we can do
* rest_requests_multi-model.ipynb
* rest_requests_single-model.ipynb 
to differentiate which notebook to use based on which way you chose to serve the model.

**Comment by MelissaFlinn** [comment]
OK - For single-model serving on Chris's cluster, I deployed the model twice - once with the models/fraud path and the other with the  models/fraud/1/model.onnx path - both were successful - so I went with the latter one in the instructions. Not sure if that was the correct decision (Chris's test drive uses the models/fraud path). He is on PTO and I did not want to pester him today
wdyt?

**Comment by erwangranger** [comment]
I think either is fine. The fuller path is probably more explicit and will feel less like magic than the shorter path. 

I'm still not sure that removing the "frozen" leading slash from our ui was the right move, because now, when I have to fill out this box, I never know if I should enter:

* `models/fraud/1/model.onnx`
* `/models/fraud/1/model.onnx`
* `/models/fraud/`
* `/models/fraud`
* `models/fraud`

and which ones are likely to work/fail. 

If 
* `/models/fraud/1/model.onnx` 

actually works, it might be the most clear, complete descriptive an unambiguous way of doing it.

**Comment by MelissaFlinn** [comment]
I just did a quick test, I can successfully deploy a model with the path: /models/fraud/1/model.onnx 
(single-model serving)

**Comment by cfchase** [comment]
> I think either is fine. The fuller path is probably more explicit and will feel less like magic than the shorter path.
> 
> I'm still not sure that removing the "frozen" leading slash from our ui was the right move, because now, when I have to fill out this box, I never know if I should enter:
> 
> * `models/fraud/1/model.onnx`
> * `/models/fraud/1/model.onnx`
> * `/models/fraud/`
> * `/models/fraud`
> * `models/fraud`
> 
> and which ones are likely to work/fail.
> 
> If
> 
> * `/models/fraud/1/model.onnx`
> 
> actually works, it might be the most clear, complete descriptive an unambiguous way of doing it.

The best path to use is always `models/fraud/` as it works for both single and multi model serving.

- Leading slashes may work, but are not normal for s3 keys.
- OpenVINO on Kserve requires a directory structure under the path of `/<version_integer>/<model_name>.onnx` or else you'll get a version error when doing the request.  They changed this for modelmesh, but not kserve.
- Using the full path to the onnx (e.g. models/fraud/1/model.onnx) will ignore any adjacent files and some onnx models have other files they need.  It would work on modelmesh for this example, but I don't think it's good in general.

**Comment by cfchase** [comment]
> I think having both options is great to have. People can choose which one they want. But the testing/validation of the endpoint will be slightly different for each, so we'll just need to add a matching notebook. we can do
> 
> * rest_requests_multi-model.ipynb
> * rest_requests_single-model.ipynb
> 
> to differentiate which notebook to use based on which way you chose to serve the model.

I agree, we should leave the old notebooks with proper naming for multi-model and add a new one for single model.

File: workshop/docs/modules/ROOT/pages/testing-the-model-api.adoc [hunk 2]
```diff
@@ -3,23 +3,21 @@
...
-image::model-serving/ds-project-model-list-name.png[Model name]
+. Take note of the model's Inference endpoint. You need this information when you test the model API.
 +
-image::model-serving/ds-project-model-list-url.png[Model URL]
+image::model-serving/ds-project-model-inference-endpoint.png[Model inference endpoint]
 
-. Return to the Jupyter environment and try out your new endpoint. You'll try REST API calls in `4_rest_requests.ipynb` and gRPC requests in `5_grpc_requests.ipynb`.
+. Return to the Jupyter environment and try out your new endpoint.
++
+If you deployed your model with multi-model serving, follow the directions in `3_rest_requests_multi_model.ipynb` to try a REST API call and ``4_grpc_requests_multi_model.ipynb` to try a gRPC API call.
```
Comment (cfchase) [inline workshop/docs/modules/ROOT/pages/testing-the-model-api.adoc:17]: you've got an extra  ` in 
```
``4_grpc_requests_multi_model.ipynb` 
```

**Review by cfchase — APPROVED** [review]

**Comment by erwangranger** [comment]
I won't have time to do the actual test, but yes, feel free to merge this in.

**Pull request merged**